util/src/main/scala/com/scalableminds/util/tools/Fox.scala

```diff
@@ -42,6 +38,11 @@ trait FoxImplicits {
 
   implicit def fox2FutureBox[T](f: Fox[T])(implicit ec: ExecutionContext): Future[Box[T]] =
     f.futureBox
+
+  // This one is no longer implicit since that has lead to confusion. Should always be used explicitly.
```

Comment:
Is it wanted that it still belongs to `FoxImplicits`?

Reply:
Yeah, that way it is available to all the classes inheriting from that Trait